crates/wasi-crypto/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "wasmtime-wasi-crypto"
+version = "0.22.0"
+authors = ["The Wasmtime Project Developers"]
+description = "Wasmtime implementation of the wasi-crypto API"
+documentation = "https://docs.rs/wasmtime-wasi-crypto"
+license = "Apache-2.0 WITH LLVM-exception"
+categories = ["wasm", "crypto"]
+keywords = ["webassembly", "wasm", "crypto"]
+repository = "https://github.com/bytecodealliance/wasmtime"
+readme = "README.md"
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+log = { version = "0.4", default-features = false }
+wasmtime = { path = "../wasmtime", version = "0.22.0", default-features = false }
+wasmtime-runtime = { path = "../runtime", version = "0.22.0" }
+wasmtime-wiggle = { path = "../wiggle/wasmtime", version = "0.22.0" }
+wasmtime-wasi = { path = "../wasi", version = "0.22.0" }
+wiggle = { path = "../wiggle", version = "0.22.0" }
+
+wasi-crypto = { path = "../wasi-crypto/spec/implementations/wasmtime", default-features = false }
+
```

Comment:
`./spec/...`?

Reply:
Sure! The relative path has been changed, thanks!